smartsim/settings/settings.py

```diff
@@ -171,7 +173,7 @@ def create_run_settings(
 
     def _detect_command(launcher: str) -> str:
         if launcher in by_launcher:
-            if launcher == "local":
+            if launcher in ["local", "dragon"]:
```

Comment:
we repeated this change in a few places. is it time to get detect command into a shared place?